platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/module/telemetry/PerformanceEvent.java

```diff
@@ -0,0 +1,68 @@
+package com.mapbox.mapboxsdk.module.telemetry;
+
+import com.mapbox.android.telemetry.Event;
+
+import android.os.Bundle;
+import android.os.Parcel;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+/**
+ * Generic Performance Event that can be used for performance measurements.
+ * Customer measurements can be added to the bundle.
+ */
+public class PerformanceEvent extends Event {
+  private static final String PERFORMANCE_TRACE = "performance.trace";
```

Comment:
Noting that this has changed to `mobile.performance_trace` and we should cut a new PR fixing it cc @osana @zugaldia @andrlee